<testng-results>
  <reporter-output>
  </reporter-output>
  <suite name="Command line suite">
    <groups>
    </groups>
    <test name="Command line test">
      <class name="com.github.srec.command.parser.xml.XmlParserTest">
        <test-method status="PASS" signature="testForm()" name="testForm" duration-ms="157" started-at="2010-08-06T20:53:51Z" finished-at="2010-08-06T20:53:51Z">
        </test-method>
        <test-method status="PASS" signature="testMethod()" name="testMethod" duration-ms="10" started-at="2010-08-06T20:53:51Z" finished-at="2010-08-06T20:53:51Z">
        </test-method>
      </class>
      <class name="com.github.srec.rec.SimpleXmlRecorderTest">
        <test-method status="PASS" signature="test()" name="test" duration-ms="536" started-at="2010-08-06T20:55:21Z" finished-at="2010-08-06T20:55:21Z">
        </test-method>
      </class>
      <class name="com.github.srec.play.PlayerTest">
        <test-method status="FAIL" signature="test()" name="test" duration-ms="15911" started-at="2010-08-06T20:53:51Z" finished-at="2010-08-06T20:54:07Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<0> but was:<1>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<0> but was:<1>
	at org.testng.Assert.fail(Assert.java:87)
	at org.testng.Assert.failNotEquals(Assert.java:441)
	at org.testng.Assert.assertEquals(Assert.java:111)
	at org.testng.Assert.assertEquals(Assert.java:326)
	at org.testng.Assert.assertEquals(Assert.java:336)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:60)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:95)
	at com.github.srec.play.PlayerTest.test(PlayerTest.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="PASS" signature="testCall()" name="testCall" duration-ms="8615" started-at="2010-08-06T20:54:07Z" finished-at="2010-08-06T20:54:15Z">
        </test-method>
        <test-method status="FAIL" signature="testError()" name="testError" duration-ms="48673" started-at="2010-08-06T20:54:15Z" finished-at="2010-08-06T20:55:04Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<19> but was:<18>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<19> but was:<18>
	at org.testng.Assert.fail(Assert.java:87)
	at org.testng.Assert.failNotEquals(Assert.java:441)
	at org.testng.Assert.assertEquals(Assert.java:111)
	at org.testng.Assert.assertEquals(Assert.java:326)
	at org.testng.Assert.assertEquals(Assert.java:336)
	at com.github.srec.play.PlayerTest.testError(PlayerTest.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="FAIL" signature="testIf()" name="testIf" duration-ms="4220" started-at="2010-08-06T20:55:04Z" finished-at="2010-08-06T20:55:08Z">
          <exception class="com.github.srec.UnsupportedFeatureException">
            <message>
              <![CDATA[org.netbeans.jemmy.JemmyException: Exception in Mouse click to get focus]]>
            </message>
            <full-stacktrace>
              <![CDATA[com.github.srec.UnsupportedFeatureException: org.netbeans.jemmy.JemmyException: Exception in Mouse click to get focus
	at com.github.srec.command.jemmy.JemmyEventCommand.internalCallMethod(JemmyEventCommand.java:32)
	at com.github.srec.command.method.MethodCommand.callMethod(MethodCommand.java:80)
	at com.github.srec.command.method.MethodCallCommand.getValue(MethodCallCommand.java:35)
	at com.github.srec.command.method.MethodCallCommand.run(MethodCallCommand.java:27)
	at com.github.srec.command.base.IfCommand.run(IfCommand.java:52)
	at com.github.srec.play.Player.play(Player.java:156)
	at com.github.srec.command.method.MethodScriptCommand.internalCallMethod(MethodScriptCommand.java:52)
	at com.github.srec.command.method.MethodCommand.callMethod(MethodCommand.java:80)
	at com.github.srec.command.method.MethodCallCommand.getValue(MethodCallCommand.java:35)
	at com.github.srec.command.method.MethodCallCommand.run(MethodCallCommand.java:27)
	at com.github.srec.command.base.IfCommand.run(IfCommand.java:52)
	at com.github.srec.play.Player.play(Player.java:156)
	at com.github.srec.play.Player.play(Player.java:145)
	at com.github.srec.play.Player.play(Player.java:119)
	at com.github.srec.play.Player.play(Player.java:85)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:55)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:95)
	at com.github.srec.play.PlayerTest.testIf(PlayerTest.java:39)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
Caused by: org.netbeans.jemmy.JemmyException: Exception in Mouse click to get focus
	at org.netbeans.jemmy.QueueTool.invokeAndWait(QueueTool.java:428)
	at org.netbeans.jemmy.QueueTool.invokeSmoothly(QueueTool.java:358)
	at org.netbeans.jemmy.drivers.focus.MouseFocusDriver.giveFocus(MouseFocusDriver.java:74)
	at org.netbeans.jemmy.drivers.text.TextKeyboardDriver.changeCaretPosition(TextKeyboardDriver.java:75)
	at org.netbeans.jemmy.drivers.text.TextKeyboardDriver.typeText(TextKeyboardDriver.java:101)
	at org.netbeans.jemmy.operators.JTextComponentOperator$4.launch(JTextComponentOperator.java:532)
	at org.netbeans.jemmy.ActionProducer.launchAction(ActionProducer.java:336)
	at org.netbeans.jemmy.ActionProducer.run(ActionProducer.java:295)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="FAIL" signature="testMethod()" name="testMethod" duration-ms="2122" started-at="2010-08-06T20:55:08Z" finished-at="2010-08-06T20:55:10Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<0> but was:<1>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<0> but was:<1>
	at org.testng.Assert.fail(Assert.java:87)
	at org.testng.Assert.failNotEquals(Assert.java:441)
	at org.testng.Assert.assertEquals(Assert.java:111)
	at org.testng.Assert.assertEquals(Assert.java:326)
	at org.testng.Assert.assertEquals(Assert.java:336)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:60)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:95)
	at com.github.srec.play.PlayerTest.testMethod(PlayerTest.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="FAIL" signature="testMethodDeep()" name="testMethodDeep" duration-ms="2251" started-at="2010-08-06T20:55:10Z" finished-at="2010-08-06T20:55:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<0> but was:<1>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<0> but was:<1>
	at org.testng.Assert.fail(Assert.java:87)
	at org.testng.Assert.failNotEquals(Assert.java:441)
	at org.testng.Assert.assertEquals(Assert.java:111)
	at org.testng.Assert.assertEquals(Assert.java:326)
	at org.testng.Assert.assertEquals(Assert.java:336)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:60)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:95)
	at com.github.srec.play.PlayerTest.testMethodDeep(PlayerTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
        <test-method status="FAIL" signature="testWhile()" name="testWhile" duration-ms="8211" started-at="2010-08-06T20:55:13Z" finished-at="2010-08-06T20:55:21Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected:<0> but was:<2>]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected:<0> but was:<2>
	at org.testng.Assert.fail(Assert.java:87)
	at org.testng.Assert.failNotEquals(Assert.java:441)
	at org.testng.Assert.assertEquals(Assert.java:111)
	at org.testng.Assert.assertEquals(Assert.java:326)
	at org.testng.Assert.assertEquals(Assert.java:336)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:60)
	at com.github.srec.testng.AbstractSRecTestNGTest.runTest(AbstractSRecTestNGTest.java:95)
	at com.github.srec.play.PlayerTest.testWhile(PlayerTest.java:43)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodHelper.invokeMethod(MethodHelper.java:644)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:546)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1002)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:137)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:121)
	at org.testng.TestRunner.runWorkers(TestRunner.java:909)
	at org.testng.TestRunner.privateRun(TestRunner.java:618)
	at org.testng.TestRunner.run(TestRunner.java:499)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:332)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:327)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:299)
	at org.testng.SuiteRunner.run(SuiteRunner.java:204)
	at org.testng.TestNG.createAndRunSuiteRunners(TestNG.java:915)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:879)
	at org.testng.TestNG.run(TestNG.java:787)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:62)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:141)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
            </full-stacktrace>
          </exception>
        </test-method>
      </class>
    </test>
  </suite>
</testng-results>
